samples/subsys/power/stm32_pm/src/main.c

```diff
@@ -0,0 +1,123 @@
+/*
+ * Copyright (c) 2016 Intel Corporation.
+ * Copyright (c) 2020 STMicroelectronics.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr.h>
+
+#include <string.h>
+#include <soc.h>
+#include <device.h>
+#include <drivers/gpio.h>
+#include <sys/printk.h>
+
+struct device *gpio_bn, *gpio_led;
+
+bool ready_to_sleep;
+
+/* in msec */
+#define SLEEP_TIME 3000
+/* setting to K_TICKS_FOREVER willl activate the deepsleep mode */
```

Comment:
This comment could be moved before the #define

Reply:
ok